[2025-10-20T16:43:35.880+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-20T16:43:35.910+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_Lab1_Rithika.serve_model_task manual__2025-10-20T16:43:25.632498+00:00 [queued]>
[2025-10-20T16:43:35.924+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_Lab1_Rithika.serve_model_task manual__2025-10-20T16:43:25.632498+00:00 [queued]>
[2025-10-20T16:43:35.926+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-20T16:43:35.948+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): serve_model_task> on 2025-10-20 16:43:25.632498+00:00
[2025-10-20T16:43:35.972+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Airflow_Lab1_Rithika', 'serve_model_task', 'manual__2025-10-20T16:43:25.632498+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/***_lab1_dag.py', '--cfg-path', '/tmp/tmp3iom12sw']
[2025-10-20T16:43:35.973+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=415) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-20T16:43:35.976+0000] {standard_task_runner.py:91} INFO - Job 24: Subtask serve_model_task
[2025-10-20T16:43:35.977+0000] {standard_task_runner.py:63} INFO - Started process 434 to run task
[2025-10-20T16:43:36.052+0000] {task_command.py:426} INFO - Running <TaskInstance: Airflow_Lab1_Rithika.serve_model_task manual__2025-10-20T16:43:25.632498+00:00 [running]> on host bddbc10ace4f
[2025-10-20T16:43:36.210+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='rithika' AIRFLOW_CTX_DAG_ID='Airflow_Lab1_Rithika' AIRFLOW_CTX_TASK_ID='serve_model_task' AIRFLOW_CTX_EXECUTION_DATE='2025-10-20T16:43:25.632498+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-20T16:43:25.632498+00:00'
[2025-10-20T16:43:36.212+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-20T16:43:36.227+0000] {logging_mixin.py:188} INFO - ðŸš€ Serving model via FastAPI simulation...
[2025-10-20T16:43:36.228+0000] {logging_mixin.py:188} INFO -    âž¤ FastAPI endpoint: /predict
[2025-10-20T16:43:36.229+0000] {logging_mixin.py:188} INFO -    âž¤ Model Path: /opt/***/working_data/model_rithika.pkl
[2025-10-20T16:43:36.230+0000] {logging_mixin.py:188} INFO - âœ… FastAPI server simulated successfully.
[2025-10-20T16:43:36.231+0000] {python.py:237} INFO - Done. Returned value was: Model served from /opt/***/working_data/model_rithika.pkl
[2025-10-20T16:43:36.232+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-20T16:43:36.260+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Airflow_Lab1_Rithika, task_id=serve_model_task, run_id=manual__2025-10-20T16:43:25.632498+00:00, execution_date=20251020T164325, start_date=20251020T164335, end_date=20251020T164336
[2025-10-20T16:43:36.314+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-20T16:43:36.341+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-20T16:43:36.346+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
